xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -49,7 +49,7 @@ namespace XBMCAddon
       if (!label2.empty())
         item->SetLabel2( label2 );
       if (!iconImage.empty())
-        item->SetIconImage( iconImage );
+        CLog::Log(LOGWARNING, "Using iconImage in ListItem constructor results in NOP. Use setArt.");
```

Comment:
@enen92 May I ask why we removed support for this parameter in the listitem ctor? while I understand that removing redundant functions makes sense I see no benefit in removing ctor arguments. Now, I have to first construct the item, then to call setArt("icon"), and maybe also even setArt("thumb"). That's one call vs. 2 or 3 calls, which I find inconvenient.

Reply:
It's deprecated since Jarvis